data-processing/entities/sentences/extractor.py

```diff
@@ -1,11 +1,15 @@
+import logging
 from dataclasses import dataclass
-from typing import Iterator
+from typing import Iterator, List
 
 import pysbd
 
 from common.parse_tex import DEFAULT_CONTEXT_SIZE, EntityExtractor, PlaintextExtractor
 from common.types import SerializableEntity
-
+# These are 'reserved characters' by the pysbd module and can potentially
+# cause issues if they are present in a string. This list was compiled from the
+# psybd source code as of 3/23/20
```

Comment:
Are all of these characters from different places in the code? I wonder if we can provide permalinks back to the locations where you found them (which I realize might be half a dozen permalinks) so someone can go back and check on whether they have changed later.

Reply:
A yes I can add those